src/lib/y2storage/guided_proposal.rb

```diff
@@ -244,10 +244,23 @@ def populated_settings
     def group_candidate_devices
       return [settings.candidate_devices] if settings.candidate_devices
 
-      candidates = disk_analyzer.candidate_disks.map(&:name)
+      candidates = reorder_candidate_disks.map(&:name)
       candidates.zip.append(candidates).uniq
     end
 
+    # Sorts candidate disk based on their type: not USB first
+    #
+    # @return [Array<Disk>]
+    def reorder_candidate_disks
```

Comment:
"sorted_candidate_disks" ? But I am fine with current name :)

Reply:
Curious, that was the first name that I chose, but sounds weird to me. So, if you are fine with the current one, I'll keep it as it.

Reply:
As mentioned, I am fine with your proposal ;)